lib/InstantSearch.js

```diff
@@ -45,6 +51,7 @@ Usage: instantsearch({
     }
 
     let client = algoliasearch(appId, apiKey);
+    client.addAlgoliaAgent('instantsearch_' + version);
```

Comment:
oh yes sure! Especially if it matches the current way we parse those informations.